hazelcast/src/test/java/com/hazelcast/multimap/LocalMultiMapStatsTest.java

```diff
@@ -140,7 +142,10 @@ public void testPutAllAndHitsGeneratedMap() {
         Map<Integer, Collection<? extends Integer>> expectedMultiMap = new HashMap<>();
         testPutAllAndHitsGeneratedTemplate(expectedMultiMap,
                 (o) -> {
-                    o.putAllAsync(expectedMultiMap);
+                    // We need to wait for putAllAsync() call to finish
+                    CompletionStage<Void> voidCompletionStage = o.putAllAsync(expectedMultiMap);
+                    CompletableFuture<Void> completableFuture = (CompletableFuture<Void>) voidCompletionStage;
+                    completableFuture.join();
```

Comment:
minor: Maybe just this would be enough
```suggestion
                    o.putAllAsync(expectedMultiMap).toCompletableFuture().join();
```

Reply:
I have used toCompletableFuture() instead of casting. Also added comment about the join() call